### PR TITLE
New package: eiwd-1.6r1

### PR DESCRIPTION
**Pull request by svenper**
#18581

### CODE REVIEW

File: srcpkgs/eiwd/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+revision=1
+wrksrc="${pkgname}-${version/r/-}"
+build_style=gnu-configure
+configure_args="--disable-systemd-service --enable-pie
+ --disable-external-ell --disable-dbus --enable-wired"
+hostmakedepends="python3-docutils pkg-config automake libtool"
+makedepends="readline-devel ell-devel>=0.31"
+depends="ell>=0.31"
+checkdepends="python3"
+short_desc="""iwd without dbus"
```
Comment (abenson) [inline srcpkgs/eiwd/template:13]: bonus ""s?

Comment (svenper) [reply srcpkgs/eiwd/template:13]: xlint workaround for lowercase initial... open to suggestions

File: srcpkgs/eiwd/template [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'eiwd'
+pkgname=eiwd
+version=1.6r1
+revision=1
+wrksrc="${pkgname}-${version/r/-}"
+build_style=gnu-configure
+configure_args="--disable-systemd-service --enable-pie
+ --disable-external-ell --disable-dbus --enable-wired"
+hostmakedepends="python3-docutils pkg-config automake libtool"
+makedepends="readline-devel ell-devel>=0.31"
```
Comment (abenson) [inline srcpkgs/eiwd/template:10]: Is the version guard necessary? I guess maybe if ell gets reverted/downgraded at some point.

Comment (svenper) [reply srcpkgs/eiwd/template:10]: At this point `ell` is broken regardless (hence `--disable-external-ell`), this is [in anticipation of fix for 1.7](https://github.com/dylanaraps/eiwd/commit/50d398e9b74f18e436e9f8d7a89e67a0333a7377)

File: srcpkgs/eiwd/files/iwd/run [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+[ -r ./conf ] && . ./conf
+exec 2>&1
```
Comment (NeelChotai) [inline srcpkgs/eiwd/files/iwd/run:3]: This line should be removed as per #21353.

Comment (Piraty) [reply srcpkgs/eiwd/files/iwd/run:3]: add a ./log/run symlink to vlogger instead

**Comment by ericonr** [comment]
PR has been inactive for a while, and so has eiwd upstream.

**Pull request closed without merging**